lib/include/robot_interface/robot_interface.h

```diff
@@ -74,6 +74,10 @@ class RobotInterface
     ros::Subscriber            _endpt_sub;
     std::vector<double>       _filt_force;
     double                    force_thres;
+    std::vector<double>      _filt_change;
+    double                  _filt_updated;
```

Comment:
This is supposed to be a time in seconds, am I right? So you should call it properly (try to take inspiration anywhere else in the code where I'm using time :+1: )